tigris/collection_test.go

```diff
@@ -305,6 +306,7 @@ func TestCollection_Search(t *testing.T) {
 			WithQuery("search query").
 			WithSearchFields("field_1").
 			WithFilter(filter.Eq("field_2", "some value")).
+			WithSorting(sort.Ascending("field_1"), sort.Descending("field_2")).
```

Comment:
Do we need to test WithSortOrder as well?

Reply:
Added in latest commit